code/src/main/java/frc/robot/RobotConfig.java

```diff
@@ -38,14 +49,24 @@
         public final int frontRightAbsoluteEncoder = 1;
         public final int rearLeftAbsoluteEncoder = 3;
         public final int rearRightAbsoluteEncoder = 2;
-        public final double swerveX = .67 / 2;
-        public final double swerveY = .25;
     }
 
     public final class Physical {
-        public final double widthInCms = 0;
-        public final double lengthInCms = 0;
-        public final double weightInKgs = 0;
+        /*
+         * When using these values for the swerve drive, make sure to divide them by two
+         * if the pivot point of the bot is in the center.
+         * 
+         * Note that standard convention is the bot is centered on the origin, facing
+         * right, along the positive x-axis. So the top-left portion of the bot is in
+         * the first quadrant, the back-left is in the second quadrant, the back-right
+         * is in the third quadrant, and the back-right is in the fourth quadrant.
+         */
+
+        // Note: these are the distances of the swerve wheels to each other, not the
+        // size of the bot's frame.
+        public final double widthInMeters = .67;
```

Comment:
IIRC this is the distance from a swerve to the center, we can check on cad today

Reply:
no, you were dividing those values by two in the previous version